### DISCUSSION TITLE
Welcome to Fastapi Users discussions!

### DISCUSSION THREAD

**Discussion by frankie567** [fastapi-users/fastapi-users#426] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**➡ If you have any question about the usage of FastAPI Users that's not clearly a bug, please open a discussion first.**

**Comment by Michaelsiriboe** [comment]
pls i am a new here will love to connect anyone who can help improve on my python skills

**Reply by Michaelsiriboe** [reply]
very nice work

**Comment by Michaelsiriboe** [comment]
pls can anyone help with my linear regression module
